src/main/java/liquibase/ext/mongodb/database/MongoLiquibaseDatabase.java

```diff
@@ -118,5 +125,19 @@ public Boolean getSupportsValidator() {
         return MongoConfiguration.SUPPORTS_VALIDATOR.getCurrentValue();
     }
 
+    @Override
```

Comment:
This should work for AWS DocumentDB as well?

Reply:
Yes, it should work. At least, it works for Localstack DynamoDB.